rust/.cargo/config.toml

```diff
@@ -1,5 +1,13 @@
 [profile.release]
-lto = true
+lto = "thin"
+codegen-units = 1
+
+[profile.release-with-debug]
+inherits = "release"
+debug = true
+# Prioritize compile time over runtime performance
+codegen-units = 16
+lto = "off"
```

Comment:
Maybe we should keep this at thin? Just to avoid CI and local producing different results?

Reply:
Done.  Good idea.  Do we still want the python builds diverging?  Or should we use thin for both there as well?

Reply:
yeah, i guess we should do the same for python.

Reply:
Done.